docs/getting_started.md

```diff
@@ -39,11 +39,14 @@ running as a Container or as a Systemd service.
 | `PROVIDE_INDEX_PAGE`                  | No        | `true`, `false`              | `false`   | Flag which returns the index page if there is one when requesting a directory.                                                                                                                                                                                                                                                                                                                                                                               |
 | `JS_TRUSTED_CERT_PATH`                | No        |                              |           | Enables the `js_fetch_trusted_certificate` directive when retrieving AWS credentials and sets the path (on the container) to the specified path                                                                                                                                                                                                                                                                                                              |
-| `HEADER_PREFIXES_TO_STRIP`            | No        |                              |           | A list of HTTP header prefixes that exclude headers client responses. List should be specified in lower-case and a semicolon (;) should be used to as a deliminator between values. For example: `x-goog-;x-something-`                                                                                                                                                                                                                                      |
+| `HEADER_PREFIXES_TO_STRIP`            | No        |                              |           | A list of HTTP header prefixes that exclude headers client responses. List should be specified in lower-case and a semicolon (;) should be used to as a deliminator between values. For example: `x-goog-;x-something-`                                                                                               |
+| `HEADER_PREFIXES_ALLOWED`            | No        |                              |           | A list of HTTP header allowed prefixes from headers client responses. List should be specified in lower-case and a semicolon (;) should be used to as a deliminator between values. For example: `x-amz-;x-something-`                                                                                                                                             |
```

Comment:
Can we please add a note that it is recommended to strip `x-amz-` for security reasons and that the gateway will do so unless `x-amz-` is explicitly provided in this configuration value?  Something like:

> A list of allowed prefixes for HTTP headers that are returned to the client in responses.  List should be specified in lower-case and a semicolon (;) should be used to as a deliminator between values. For example: `x-amz-;x-something-`. It is **NOT** recommended to return `x-amz-` headers for security reasons. Think carefully about what is allowed here.

And since the other one has a typo originally I can suggest this text:
> A list of HTTP header prefixes that exclude headers from client responses. List should be specified in lower-case and a semicolon (;) should be used to as a deliminator between values. For example: `x-goog-;x-something-`. Headers starting with `x-amz-` will be stripped by default for security reasons unless explicitly added in `HEADER_PREFIXES_ALLOWED`.

Reply:
Changed